test/integration/index.js

```diff
@@ -75,8 +78,26 @@ utils.assert(
   'App instances do not match.'
 );
 
+function updateRules() {
+  // Update database rules to the defaults. Rest of the test suite
+  // expects it.
+  const client = new apiRequest.SignedApiRequestHandler(defaultApp);
+  const dbUrl =  url.parse(defaultApp.options.databaseURL);
+  const defaultRules = {
+    rules : {
+      '.read': 'auth != null',
+      '.write': 'auth != null',
+    },
+  };
+  const headers = {
+    'Content-Type': 'application/json',
+  };
+  return client.sendRequest(dbUrl.host, 443, '/.settings/rules.json', 
+    'PUT', defaultRules, headers, 10000);
```

Comment:
It does, but that is ok. Other SDKs do the same during integration tests. We also advise developers to use a separate, dedicated project for integration testing since the test suite modifies both the content and the rules of the database.